src/python/test/test_sklearn_rips_persistence.py

```diff
@@ -10,6 +10,12 @@
 
 from gudhi.datasets.generators import points
 from gudhi.sklearn.rips_persistence import RipsPersistence
+from gudhi import RipsComplex, SimplexTree
+from gudhi._ripser import _lower, _full, _sparse, _lower_to_coo, _lower_cone_radius
+from gudhi import bottleneck_distance
```

Comment:
If you import bottleneckdistance (and not the hera version), CGAL is required for this test (but not for the module).
Ah... forget this point, CGAL is already required because of `gudhi.datasets.generators`.

Reply:
Ah, it is good that you thought about it, because it didn't cross my mind at all.
Indeed, CGAL is already required for this particular test. Ideally it would be better if we didn't gratuitously require CGAL to test things that don't actually use CGAL, but I guess it is ok for a test.